pages/_admin/index.tsx

```diff
@@ -100,12 +119,13 @@ export async function getServerSideProps({ req, query }) {
   const session = await getSession({ req })
 
   const deleted = query.deleted ? true : false
```

Comment:
Technically both of these could be simplified to be `!!query.deleted` and `!!query.approved` but I personally find that syntax super confusing when I read it so it probably makes sense to leave it as a ternary!

Reply:
How about just `query.deleted === true`? Maybe the clearest and most direct option of all?
As general rule, ternaries with `true : false` can always be converted to a strict equals comparison to true or false.